docs/src/http_api_guide.md

```
@@ -0,0 +1,411 @@
+| ------ | ------ | ----------------------------------- |
+| hashes | string | Comma-separated hex hashes of UTXOs |
+
+#### Description
+Returns mined info (e.g., block height, inclusion) for the specified UTXO hashes in an array.
+
+#### Response
+
+| Name                          | Type       | Description                                                                |
+| ----------------------------- | ---------- | -------------------------------------------------------------------------- |
+| utxos                         | array      | List of mined UTXO information                                             |
+| `utxos[].utxo_hash `          | `byte[32]` | The hash of the UTXO (in byte array format; must be hex-encoded for reuse) |
```

markdownlint-cli2 (0.17.2)

Spaces inside code span elements

(MD038, no-space-in-code)